ananicy.d/00-types.types

```diff
@@ -90,7 +94,7 @@
 
 {"type":"VPN"}
 
-{"type":"VM"}
+{"type":"VM,CPU_H,IO_H",    "ionice":7}
```

Comment:
The `_H` postfix stands for higher priority. If we are to create a new type called `VM,CPU_H,IO_H`, that would imply that IO nice is about 0, and that nice is about -20.
If are to keep the `"ionice":7`, then we would need to change the type to `VM,CPU_U,IO_L` or `VM,IO_L`.

Reply:
I don't know, I didn't see documentation that says anything about